php/sw-fw-less/composer.json

```diff
@@ -21,5 +21,8 @@
       "App\\": "app/"
     }
   },
+  "config": {
+    "optimize-autoloader": true
```

Comment:
not need, this will be done for all php frameworks (using the command line)

Reply:
Ok. It has been removed.